lib/situation_room/sites/site_controller.ex

```diff
@@ -66,6 +66,8 @@ defmodule SituationRoom.Site.Controller do
     end
   end
 
+  forward("/:id/checks", to: SituationRoom.Site.Check.Controller)
```

Comment:
I think this change should be included in the commit where the Controller is added.  Or the adding of the controller should move forward to this commit here.  There is not value in having the controller introduced but not consumed.  That breaks the atomic nature a bit.